src/config/chain.d.ts

```diff
@@ -17,6 +17,7 @@ export const CHAIN_ID: Record<ChainName, ChainId> = {
 
 // Values match that required of onboard and returned by CGW
 export enum WALLETS {
+  MOBILE_SAFE = 'mobileSafe',
```

Comment:
```suggestion
  SAFE_MOBILE= 'safeMobile',
```

The pairing module is called 'Mobile Safe' and the [name is referenced](https://github.com/gnosis/safe-react/blob/84ae5541ff6fe4d3647eee0e51e856fe57c9303d/src/logic/wallets/onboard.ts#L118) when determining if the same Onboard instance can persist across chain switching (see [here](https://github.com/gnosis/safe-react/blob/84ae5541ff6fe4d3647eee0e51e856fe57c9303d/src/logic/wallets/utils/walletList.ts#L71)).

These should match as closely as possible otherwise the pairing connection will interrupt when switching chain.